### PR TITLE
[ngspice] Update to release 35

### PR DESCRIPTION
**Pull request by marekr**
- #### What does your PR fix?  
Updates to latest r35 release

Also fixes the project's vcproj dumping files in C:\ root

- #### Which triplets are supported/not supported? Have you updated the [CI baseline]
Windows x64, x86 only

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  

Yes its committed.

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: ports/ngspice/portfile.cmake [hunk 1]
```diff
@@ -8,12 +8,14 @@ set(VCPKG_CRT_LINKAGE static)
...
-    FILENAME "ngspice-34.tar.gz"
-    SHA512 5e90727f3f6b8675b83f71e1961d33cd498081a7f3bea5d081521f12ecb3979775159f083f84a5856233529505262c399f75d305758af51894a1245603476cf8
+    REF 35
+    FILENAME "ngspice-35.tar.gz"
+    SHA512 2f9b0f951e3ca8d52692beadb895b352311f67b8760f99d0e2f4718fce4b497dd68e2b933029eeacb4ed57551e959bc6e3747e64feb4722a4f841e734f5a664b
     PATCHES
         use-winbison-sharedspice.patch
         use-winbison-vngspice.patch
+        remove-post-build.patch
+		remove-64-in-codemodel-name.patch
```
Comment (JonLiu1993) [inline ports/ngspice/portfile.cmake:18]:
```suggestion
        remove-post-build.patch
        remove-64-in-codemodel-name.patch

```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Comment by marekr** [comment]
Updated with the suggested change and x-add-version

**Review by JonLiu1993 — APPROVED** [review]

**Comment by JonLiu1993** [comment]
Feature codemodels tested successfully in the following triplet:
- x86-windows
- x64-windows

**Comment by ras0219-msft** [comment]
LGTM, thanks for the update!

**Pull request merged**